lang/src/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransform.kt

```diff
@@ -151,22 +291,32 @@ private object FromSourceToBexpr : PartiqlAst.FromSource.Converter<PartiqlLogica
         }
     }
 
-    override fun convertUnpivot(node: PartiqlAst.FromSource.Unpivot): PartiqlLogical.Bexpr {
-        TODO("Support for UNPIVOT")
-    }
+    override fun convertUnpivot(node: PartiqlAst.FromSource.Unpivot): PartiqlLogical.Bexpr =
+        INVALID_BEXPR.also { problemHandler.handleUnimplementedFeature(node, "UNPIVOT") }
```

Comment:
Just a bit wondering. Are the changes here also related to PartiQL DML?

Reply:
Not directly, but one of my goals in this PR was to get all of the error handling to be completed.  Since there was other error handling in this file that was (such as the one you point out here) that was kinda incomplete I just to do it for the entire class.